src/main/webapp/file.xhtml

```diff
@@ -222,51 +223,95 @@
-                                                        <a jsf:id="endNoteLink" jsf:action="#{FilePage.fileDownloadService.downloadDatafileCitationXML(FilePage.fileMetadata)}" >
+                                                        <a jsf:id="endNoteLink" jsf:action="#{FilePage.fileDownloadService.downloadDatasetCitationXML(FilePage.fileMetadata.datasetVersion.dataset)}" >
                                                             #{bundle['dataset.cite.downloadBtn.xml']}
                                                         </a>
                                                     </li>
                                                     <li>
-                                                        <a jsf:id="risLink" jsf:actionListener="#{FilePage.fileDownloadService.downloadDatafileCitationRIS(FilePage.fileMetadata)}">
+                                                        <a jsf:id="risLink" jsf:actionListener="#{FilePage.fileDownloadService.downloadDatasetCitationRIS(FilePage.fileMetadata.datasetVersion.dataset)}">
                                                             #{bundle['dataset.cite.downloadBtn.ris']}
                                                         </a>
                                                     </li>
                                                     <li>
-                                                        <a jsf:id="bibLink" jsf:actionListener="#{FilePage.fileDownloadService.downloadDatafileCitationBibtex(FilePage.fileMetadata)}" target="_blank">
+                                                        <a jsf:id="bibLink" jsf:actionListener="#{FilePage.fileDownloadService.downloadDatasetCitationBibtex(FilePage.fileMetadata.datasetVersion.dataset)}" target="_blank">
                                                             #{bundle['dataset.cite.downloadBtn.bib']}
                                                         </a>
                                                     </li>
                                                 </ul>
                                             </div>
                                             <div id="citation-standards" class="small text-muted text-right"><span class="glyphicon glyphicon-info-sign"></span> <h:outputText value=" #{bundle['dataset.cite.standards.tip']}" escape="false"/></div>
                                         </div>
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="col-xs-12">
+                                <div class="alert alert-warning" style="margin:0;padding:0 .5em;background:none;border-radius:0;border:0;color:#333;">
+                                  <h:outputFormat value="#{bundle['file.citation.datafile']}" escape="false" />
+                                </div>
+                            </div>
+                            <div id="citation-block-2" class="col-xs-12">
```

Comment:
While re-using the ID and adding a "-2" to the end solves the duplicate ID warnings when validating, it breaks the CSS being applied to the original ID. I am working on a commit to resolve these minor layout issues, and maybe refactor some of the original CSS code to make it more scalable.